src/main/java/com/google/step/servlets/UserServlet.java

```diff
@@ -0,0 +1,47 @@
+package com.google.step.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.step.datamanager.UserManager;
+import com.google.step.datamanager.UserManagerDatastore;
+import com.google.step.model.Deal;
+import com.google.step.model.Restaurant;
+import com.google.step.model.Tag;
+import com.google.step.model.User;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles users. */
+@WebServlet("/api/users/*")
+public class UserServlet extends HttpServlet {
+
+  private UserManager userManager = new UserManagerDatastore();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json");
+
+    String idString = request.getPathInfo().substring(1); // Remove '/'
+    long id = Long.parseLong(idString);
```

Comment:
might need to try catch NumberFormatException if the parsing fails

Reply:
oh I do this in the next PR. I'll move it here.